app/speak_slack.rb

```diff
@@ -0,0 +1,15 @@
+# Takes an array of food truck names and formats it to slack
+require_relative '../constants'
+
+module FoodTruck
+  class SpeakSlack
+    def self.trucks(trucks: [])
```

Comment:
`trucks` doesn't communicate to me what this method is for.  It looks like maybe it's for formatting?  Maybe `FoodTruck::SlackFormatter.format(trucks)` (or similar) would communicate its purpose better?

Also, why the default value for the `trucks` arg?  Does it make sense to allow callers to call this without providing that argument?